plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java

```diff
@@ -33,6 +34,7 @@ public TestingClickHouseServer()
     {
         // Use 2nd stable version
         dockerContainer = (ClickHouseContainer) new ClickHouseContainer(CLICKHOUSE_IMAGE)
+                .withCopyFileToContainer(forClasspathResource("config.xml"), "/etc/clickhouse-server/config.d/config.xml")
```

Comment:
Is there /etc/clickhouse-server/config.d/config.xml in the image?
or we're creating a new file? then, can we give it a more specific name?

Reply:
It's a new file. I renamed to `custom.yml`.